gobblin-core/src/main/java/gobblin/source/extractor/extract/jdbc/JdbcExtractor.java

```diff
@@ -317,11 +317,13 @@ public void extractMetadata(String schema, String entity, WorkUnit workUnit) thr
           JsonObject jsonObject = gson.fromJson(jsonStr, JsonObject.class).getAsJsonObject();
           targetSchema.add(jsonObject);
           headerColumns.add(targetColumnName);
+          sourceColumnName = "`" + sourceColumnName + "`";
```

Comment:
Yes, backtick is for mysql. Oracle uses double quote and the content within double quotes is case sensitive, so in most cases the column name should be uppercased before enclosing in quotes. SQLServer uses square brackets.